etc/kayobe/kolla/globals.yml

```diff
@@ -69,5 +69,131 @@ prometheus_ceph_mgr_exporter_endpoints:
 {% endfor %}
 {% endif %}
 
-#############################################################################
+
+########################################
+# Prometheus blackbox monitoring options
+########################################
+{% raw %}
+kolla_internal_endpoint: "{{ internal_protocol }}://{{ kolla_internal_fqdn | put_address_in_context('url') }}"
+kolla_public_endpoint: "{{ public_protocol }}://{{ kolla_external_fqdn | put_address_in_context('url') }}"
```

Comment:
This won't work with a single frontend port (i.e. hostname-based URLs).

Reply:
I've gone with the same format as the upstream version, and included definitions for any missing endpoints